apps/jsonrpc2/lib/jsonrpc2/response/block.ex

```diff
@@ -28,7 +29,32 @@ defmodule JSONRPC2.Response.Block do
     :uncles
   ]
 
-  def new(internal_block) do
+  @type transactions :: [binary()] | [ResponseTransaction.t()]
+
+  @type t :: %__MODULE__{
+          number: binary(),
+          hash: binary(),
+          parentHash: binary(),
+          nonce: binary(),
+          sha3Uncles: binary(),
+          logsBloom: binary(),
+          transactionsRoot: binary(),
+          stateRoot: binary(),
+          receiptsRoot: binary(),
+          miner: binary(),
+          difficulty: binary(),
+          totalDifficulty: binary(),
+          extraData: binary(),
+          size: binary(),
+          gasLimit: binary(),
+          gasUsed: binary(),
+          timestamp: binary(),
+          transactions: transactions(),
+          uncles: []
+        }
+
+  @spec new(Block.t(), boolean()) :: t()
+  def new(internal_block, full_transactions) do
```

Comment:
Would it be useful for `full_transactions` to have a default? Maybe `false` for backward compatibility?